test/e2e/storage/testsuites/multivolume.go

```diff
@@ -328,19 +328,54 @@ func (t *multiVolumeTestSuite) DefineTests(driver storageframework.TestDriver, p
 		if pattern.SnapshotType == "" {
 			e2eskipper.Skipf("Driver %q does not support snapshots - skipping", dInfo.Name)
 		}
-		if pattern.VolMode == v1.PersistentVolumeBlock {
-			// TODO: refactor prepareSnapshotDataSourceForProvisioning() below to use
-			// utils.CheckWriteToPath / utils.CheckReadFromPath and remove
-			// redundant InjectContent(). This will enable block volume tests.
-			e2eskipper.Skipf("This test does not support block volumes -- skipping")
-		}
+		
+		// if pattern.VolMode == v1.PersistentVolumeBlock {
+		// 	// TODO: refactor prepareSnapshotDataSourceForProvisioning() below to use
+		// 	// utils.CheckWriteToPath / utils.CheckReadFromPath and remove
+		// 	// redundant InjectContent(). This will enable block volume tests.
+		// 	e2eskipper.Skipf("This test does not support block volumes -- skipping")
+		// }
 
 		// Create a volume
 		testVolumeSizeRange := t.GetTestSuiteInfo().SupportedSizeRange
 		resource := storageframework.CreateVolumeResource(l.driver, l.config, pattern, testVolumeSizeRange)
 		l.resources = append(l.resources, resource)
 		pvcs := []*v1.PersistentVolumeClaim{resource.Pvc}
 
+
+		// Creating injector pod
+		ginkgo.By(fmt.Sprintf("Creating injector pod for volume"))
```

Comment:
probably these snippet that creates the file on both a block and a filesystem volume should be inside `prepareSnapshotDataSourceForProvisioning` as the it says in the TODO that @jsafrane added

> 			// TODO: refactor prepareSnapshotDataSourceForProvisioning() below to use
>			// utils.CheckWriteToPath / utils.CheckReadFromPath and remove
>			// redundant InjectContent(). This will enable block volume tests.

Reply:
I completely agree Mau. Thanks for reviewing, I've removed this piece of code and went with the approach we discussed over in Slack